src/current/_includes/sidebar-all-releases.json

```diff
@@ -1,25 +1,19 @@
 {%- assign versions = site.data.versions | where_exp: "versions", "versions.major_version != site.versions['dev']" | where_exp: "versions", "versions.major_version != site.versions['stable']" | map: "major_version" -%}
```

Comment:
Noting that this file is not used by v24.1's nav. I think you should revert this change because older versions need this file to be the same.